From: Elimar Riesebieter <riesebie@lxtec.de>
Subject: Clean build system for running a DFSG build.
Last-Update: 2010-05-23

Prevent to not configure cs46xx, korg1212, maestro3, ymfpci. These drivers do
have binary code in sorce.

Index: alsa-driver-1.0.25/acinclude.m4
===================================================================
--- alsa-driver-1.0.25.orig/acinclude.m4	2012-01-25 20:29:17.000000000 +1100
+++ alsa-driver-1.0.25/acinclude.m4	2012-02-08 20:21:48.558528741 +1100
@@ -151,8 +151,6 @@
 	CONFIG_SND_OXYGEN_LIB=""
 	CONFIG_SND_OXYGEN=""
 	CONFIG_SND_CS4281=""
-	CONFIG_SND_CS46XX=""
-	CONFIG_SND_CS46XX_NEW_DSP=""
 	CONFIG_SND_CS5530=""
 	CONFIG_SND_CS5535AUDIO=""
 	CONFIG_SND_CTXFI=""
@@ -207,11 +205,8 @@
 	CONFIG_SND_ICE1724=""
 	CONFIG_SND_INTEL8X0=""
 	CONFIG_SND_INTEL8X0M=""
-	CONFIG_SND_KORG1212=""
 	CONFIG_SND_LOLA=""
 	CONFIG_SND_LX6464ES=""
-	CONFIG_SND_MAESTRO3=""
-	CONFIG_SND_MAESTRO3_INPUT=""
 	CONFIG_SND_MIXART=""
 	CONFIG_SND_NM256=""
 	CONFIG_SND_PCXHR=""
@@ -227,7 +222,6 @@
 	CONFIG_SND_VIA82XX_MODEM=""
 	CONFIG_SND_VIRTUOSO=""
 	CONFIG_SND_VX222=""
-	CONFIG_SND_YMFPCI=""
 	CONFIG_SND_PDPLUS=""
 	CONFIG_SND_PPC=""
 	CONFIG_SND_POWERMAC=""
@@ -763,16 +757,16 @@
   [                          pc98-cs4232, fm801, es1968, ad1889, als300, ]
   [                          als4000, ali5451, asihpi, atiixp, atiixp-modem, ]
   [                          au8810, au8820, au8830, aw2, azt3328, bt87x, ]
-  [                          ca0106, cmipci, oxygen, cs4281, cs46xx, cs5530, ]
+  [                          ca0106, cmipci, oxygen, cs4281, cs5530, ]
   [                          cs5535audio, ctxfi, darla20, gina20, layla20, ]
   [                          darla24, gina24, layla24, mona, mia, echo3g, ]
   [                          indigo, indigoio, indigodj, indigoiox, indigodjx, ]
   [                          emu10k1x, ens1370, ens1371, es1938, hda-intel, ]
   [                          hdsp, hdspm, ice1712, ice1724, intel8x0, ]
-  [                          intel8x0m, korg1212, lola, lx6464es, maestro3, ]
+  [                          intel8x0m, lola, lx6464es, ]
   [                          mixart, nm256, pcxhr, riptide, rme32, rme96, ]
   [                          rme9652, sis7019, sonicvibes, trident, via82xx, ]
-  [                          via82xx-modem, virtuoso, vx222, ymfpci, pdplus, ]
+  [                          via82xx-modem, virtuoso, vx222, pdplus, ]
   [                          powermac, ps3, aoa, aoa-fabric-layout, aoa-onyx, ]
   [                          aoa-tas, aoa-toonie, aoa-soundbus, ]
   [                          aoa-soundbus-i2s, armaaci, pxa2xx-ac97, ]
@@ -847,7 +841,7 @@
   [                          seq-hrtimer-default, seq-rtctimer-default, ]
   [                          support-old-api, pcm-xrun-debug, ac97-power-save, ]
   [                          sb16-csp, fm801-tea575x-bool, es1968-radio, ]
-  [                          bt87x-overclock, cs46xx-new-dsp, es1968-input, ]
+  [                          bt87x-overclock, es1968-input, ]
   [                          hda-hwdep, hda-reconfig, hda-input-beep, ]
   [                          hda-input-jack, hda-patch-loader, ]
   [                          hda-codec-realtek, hda-enable-realtek-quirks, ]
@@ -856,7 +850,7 @@
   [                          hda-codec-conexant, hda-codec-ca0110, ]
   [                          hda-codec-ca0132, hda-codec-cmedia, ]
   [                          hda-codec-si3054, hda-generic, hda-power-save, ]
-  [                          maestro3-input, powermac-auto-drc, ]
+  [                          powermac-auto-drc, ]
   [                          usb-caiaq-input, bf5xx-mmap-support, ]
   [                          bf5xx-multichan-support, bf5xx-have-cold-reset, ]
   [                          dm365-aic3x-codec, dm365-voice-codec, ]
@@ -1850,25 +1844,6 @@
       CONFIG_SND_CS4281="m"
     fi
   fi
-  if alsa_check_kconfig_card "cs46xx"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
-      CONFIG_SND_TIMER="m"
-      CONFIG_SND_PCM="m"
-      test $kvernum -ge 206014 && CONFIG_AC97_BUS="m"
-      CONFIG_SND_VMASTER="y"
-      CONFIG_SND_RAWMIDI="m"
-      CONFIG_SND_AC97_CODEC="m"
-      CONFIG_SND_CS46XX="m"
-    fi
-  fi
-  if alsa_check_kconfig_option "cs46xx-new-dsp"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ) &&
-      ( test "$CONFIG_SND_CS46XX" = "y" -o "$CONFIG_SND_CS46XX" = "m" ); then
-      CONFIG_SND_CS46XX_NEW_DSP="y"
-    fi
-  fi
   if alsa_check_kconfig_card "cs5530"; then
     if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
       ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ) &&
@@ -2345,14 +2320,6 @@
       CONFIG_SND_INTEL8X0M="m"
     fi
   fi
-  if alsa_check_kconfig_card "korg1212"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
-      CONFIG_SND_TIMER="m"
-      CONFIG_SND_PCM="m"
-      CONFIG_SND_KORG1212="m"
-    fi
-  fi
   if alsa_check_kconfig_card "lola"; then
     if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
       ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
@@ -2370,26 +2337,6 @@
       CONFIG_SND_LX6464ES="m"
     fi
   fi
-  if alsa_check_kconfig_card "maestro3"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
-      CONFIG_SND_TIMER="m"
-      CONFIG_SND_PCM="m"
-      test $kvernum -ge 206014 && CONFIG_AC97_BUS="m"
-      CONFIG_SND_VMASTER="y"
-      CONFIG_SND_AC97_CODEC="m"
-      CONFIG_SND_MAESTRO3="m"
-    fi
-  fi
-  if alsa_check_kconfig_option "maestro3-input"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ) &&
-      ( test "$CONFIG_SND_MAESTRO3" = "y" -o "$CONFIG_SND_MAESTRO3" = "m" ) &&
-      ( test "$CONFIG_INPUT" = "y" -o "$CONFIG_INPUT" = "m" ) &&
-      ( test $kvernum -ge 206023 ); then
-      CONFIG_SND_MAESTRO3_INPUT="y"
-    fi
-  fi
   if alsa_check_kconfig_card "mixart"; then
     if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
       ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
@@ -2548,21 +2495,6 @@
       CONFIG_SND_VX222="m"
     fi
   fi
-  if alsa_check_kconfig_card "ymfpci"; then
-    if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
-      ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
-      CONFIG_SND_TIMER="m"
-      CONFIG_SND_HWDEP="m"
-      CONFIG_SND_RAWMIDI="m"
-      CONFIG_SND_PCM="m"
-      test $kvernum -ge 206014 && CONFIG_AC97_BUS="m"
-      CONFIG_SND_VMASTER="y"
-      CONFIG_SND_OPL3_LIB="m"
-      CONFIG_SND_MPU401_UART="m"
-      CONFIG_SND_AC97_CODEC="m"
-      CONFIG_SND_YMFPCI="m"
-    fi
-  fi
   if alsa_check_kconfig_card "pdplus"; then
     if  ! ( test "$CONFIG_UML" = "y" -o "$CONFIG_UML" = "m" ) &&
       ( test "$CONFIG_SND_PCI" = "y" -o "$CONFIG_SND_PCI" = "m" ); then
@@ -4819,12 +4751,6 @@
 if test -n "$CONFIG_SND_CS4281"; then
   AC_DEFINE(CONFIG_SND_CS4281_MODULE)
 fi
-if test -n "$CONFIG_SND_CS46XX"; then
-  AC_DEFINE(CONFIG_SND_CS46XX_MODULE)
-fi
-if test -n "$CONFIG_SND_CS46XX_NEW_DSP"; then
-  AC_DEFINE(CONFIG_SND_CS46XX_NEW_DSP)
-fi
 if test -n "$CONFIG_SND_CS5530"; then
   AC_DEFINE(CONFIG_SND_CS5530_MODULE)
 fi
@@ -4978,21 +4904,12 @@
 if test -n "$CONFIG_SND_INTEL8X0M"; then
   AC_DEFINE(CONFIG_SND_INTEL8X0M_MODULE)
 fi
-if test -n "$CONFIG_SND_KORG1212"; then
-  AC_DEFINE(CONFIG_SND_KORG1212_MODULE)
-fi
 if test -n "$CONFIG_SND_LOLA"; then
   AC_DEFINE(CONFIG_SND_LOLA_MODULE)
 fi
 if test -n "$CONFIG_SND_LX6464ES"; then
   AC_DEFINE(CONFIG_SND_LX6464ES_MODULE)
 fi
-if test -n "$CONFIG_SND_MAESTRO3"; then
-  AC_DEFINE(CONFIG_SND_MAESTRO3_MODULE)
-fi
-if test -n "$CONFIG_SND_MAESTRO3_INPUT"; then
-  AC_DEFINE(CONFIG_SND_MAESTRO3_INPUT)
-fi
 if test -n "$CONFIG_SND_MIXART"; then
   AC_DEFINE(CONFIG_SND_MIXART_MODULE)
 fi
@@ -5035,9 +4952,6 @@
 if test -n "$CONFIG_SND_VX222"; then
   AC_DEFINE(CONFIG_SND_VX222_MODULE)
 fi
-if test -n "$CONFIG_SND_YMFPCI"; then
-  AC_DEFINE(CONFIG_SND_YMFPCI_MODULE)
-fi
 if test -n "$CONFIG_SND_PDPLUS"; then
   AC_DEFINE(CONFIG_SND_PDPLUS_MODULE)
 fi
@@ -6194,8 +6108,6 @@
 AC_SUBST(CONFIG_SND_OXYGEN_LIB)
 AC_SUBST(CONFIG_SND_OXYGEN)
 AC_SUBST(CONFIG_SND_CS4281)
-AC_SUBST(CONFIG_SND_CS46XX)
-AC_SUBST(CONFIG_SND_CS46XX_NEW_DSP)
 AC_SUBST(CONFIG_SND_CS5530)
 AC_SUBST(CONFIG_SND_CS5535AUDIO)
 AC_SUBST(CONFIG_SND_CTXFI)
@@ -6250,11 +6162,8 @@
 AC_SUBST(CONFIG_SND_ICE1724)
 AC_SUBST(CONFIG_SND_INTEL8X0)
 AC_SUBST(CONFIG_SND_INTEL8X0M)
-AC_SUBST(CONFIG_SND_KORG1212)
 AC_SUBST(CONFIG_SND_LOLA)
 AC_SUBST(CONFIG_SND_LX6464ES)
-AC_SUBST(CONFIG_SND_MAESTRO3)
-AC_SUBST(CONFIG_SND_MAESTRO3_INPUT)
 AC_SUBST(CONFIG_SND_MIXART)
 AC_SUBST(CONFIG_SND_NM256)
 AC_SUBST(CONFIG_SND_PCXHR)
@@ -6270,7 +6179,6 @@
 AC_SUBST(CONFIG_SND_VIA82XX_MODEM)
 AC_SUBST(CONFIG_SND_VIRTUOSO)
 AC_SUBST(CONFIG_SND_VX222)
-AC_SUBST(CONFIG_SND_YMFPCI)
 AC_SUBST(CONFIG_SND_PDPLUS)
 AC_SUBST(CONFIG_SND_PPC)
 AC_SUBST(CONFIG_SND_POWERMAC)
